File: shared/schemas/python/address.py
"""
Address Entity Models
"""

from typing import Optional

from .base import BaseEntity


class Address(BaseEntity):
    """Address entity model"""

    address_id: str
    address_line_1: str
    address_line_2: Optional[str] = None
    city: str
    state_region: str
    postal_code: int
    country: str

File: shared/schemas/typescript/address.ts
/**
 * Address Entity Types
 */

import { BaseEntity } from './base'

export interface Address extends BaseEntity {
  address_id: string
  address_line_1: string
  address_line_2?: string
  city: string
  state_region: string
  postal_code: number
  country: string
} 